parser_library/src/processing/preprocessors/preprocessor_utils.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+#include "preprocessor_utils.h"
+
+#include "lexing/logical_line.h"
+#include "semantics/statement.h"
+
+namespace hlasm_plugin::parser_library::processing {
+
+namespace {
+std::vector<semantics::preproc_details::name_range> get_operands_list(
```

Comment:
- are multiline operands a concern here?
- maybe make it a public function and add a few unit tests?
- the functions seems to expect a pre-validated input, is that the case? If so, just add a comment.

Reply:
- Behavior added
- Tests added
- Comment added